src/librustdoc/html/render/mod.rs

```diff
@@ -667,25 +667,35 @@ fn short_item_info(
 
     // Render unstable items. But don't render "rustc_private" crates (internal compiler crates).
     // Those crates are permanently unstable so it makes no sense to render "unstable" everywhere.
-    if let Some((StabilityLevel::Unstable { reason: _, issue, .. }, feature)) = item
-        .stability(cx.tcx())
-        .as_ref()
-        .filter(|stab| stab.feature != sym::rustc_private)
-        .map(|stab| (stab.level, stab.feature))
+    if let Some(rustc_attr::Stability {
+        level: StabilityLevel::Unstable { unstables, .. }, ..
+    }) = item.stability(cx.tcx())
     {
         let mut message =
             "<span class=\"emoji\">🔬</span> This is a nightly-only experimental API.".to_owned();
```

Comment:
This string and the parentheses could be incorporated to the `extra_info.push` below.